### PR TITLE
update Readme with correct credential setup instructions #84

### LINKED ISSUE #84: Readme has wrong instructions on where to obtain App Id Credentials from
### Background on the problem the feature will solve/improved user experience

Readme states the credentials should be obtained from the Service Credential tab, however, using credentials from that tab results in the error message: {"error":"invalid_client","error_description":"response_mode : web_message is supported only for single page applications"}

### Describe the solution you'd like

Credentials should be obtained from the Applications tab.
Update readme with the screenshot below. 

creds

### Acceptance Criteria

Developer is able to successfully authenticate

### PR DESCRIPTION
**Pull request by hcemk**
Contributes to #85 

## What did you do?
Fixed instructions on where to get the credentials from.

## Why did you do it?
Instructions were broken. Discussed the fix with the team.

## How have you tested it?
Yes

## Were docs updated if needed?

- [ ] No
- [x] Yes
- [ ] N/A

#### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new console logs
- [x] Fixes entire issue
- [ ] Partial fix for issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,11 +55,11 @@ Fill out the App ID credentials in `web/src/app/services/appid-authentication.se
 
 ![](images/spa_instruction.png)
 
-2. Register `http://localhost:4200/*` or other redirect_uri you trust to a list of authorized clients.
+2. Register `http://localhost:4200/*` or other redirect_uri you trust to a list of authorized clients. This could be the uri your app uses, eg, `https://localhost:8080/*`
```
Comment (upkarlidder) [inline README.md]: I would say `should be` and not could be. This is a required step. @hcemk can you please revise the wording?

**Comment by upkarlidder** [comment]
@hcemk I asked for a small change. Additionally, you need to fix the DCO part. Please follow the instructions by clicking on `Details` under the failed DCO test.

image

**Comment by hcemk** [comment]
@upkarlidder comments addressed, please review

**Review by upkarlidder — APPROVED** [review]
Thank you @hcemk for your contribution!

**Pull request merged**